plugins/pm2/_pm2

```diff
@@ -139,7 +139,7 @@ case "$words[1]" in
     stop|restart|delete|reload|reset)
         _arguments $id_all_comp && return 0
         ;;
-    env|inspect|monitor|unmonitor|discribe)
+    env|inspect|monitor|unmonitor|describe)
```

Comment:
Afacit the current code is buggy: https://futurestud.io/tutorials/pm2-list-processes-and-show-process-details#processdetails